x-pack/plugin/src/test/resources/rest-api-spec/test/data_frame/preview_transforms.yml

```diff
@@ -203,6 +203,29 @@ setup:
               }
             }
           }
+  - do:
+      catch: /field \[super_metric\] cannot be both an object and a field/
+      data_frame.preview_data_frame_transform:
+        body: >
+          {
+            "source": { "index": "airline-data" },
+            "pivot": {
+              "group_by": {
+                "time": {"date_histogram": {"fixed_interval": "1h", "field": "time", "format": "yyyy-MM-DD HH"}}},
```

Comment:
`DD` is creeping back into this file after it was corrected to `dd` in some places.  We can deal with this in another PR though, maybe by removing `format` altogether.  But please don't propagate `DD` anywhere else.

Reply:
definitely, let me remove that stuff.